sdk/trts/linux/ctd.c

```diff
@@ -0,0 +1,1334 @@
+/**
+ * @file dis_main.c
```

Comment:
Can we cleanup the header declaration and add the license?

Reply:
Fixed in the new force-push